app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import Any
+
+
+def copy_file(command: str) -> Any:
+    if len(command.split()) != 3 or command.split()[0] != "cp":
+        print("Invalid command format."
+              " Use: cp <source_file> <destination_file>")
+        return
+
+    cp, old_file, new_file = command.split()
```

Comment:
now you use `command.split()` three times, try to optimize and leave only one use `command.split()`